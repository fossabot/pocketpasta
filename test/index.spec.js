// const assert = require('assert');

// describe('pocketpasta.com page', () => {
//   it('should have the right title', () => {
//     browser.url('https://pocketpasta.com');
//     const title = browser.getTitle();
//     assert.strict.equal(title, 'pocketpasta');
//   });
// });

import { shallowMount } from '@vue/test-utils';
import index from '@/pages/index.vue';

const factory = () =>
  shallowMount(index, {
    propsData: {
      label: 'Job Title',
      value: 'Full Stack Web Developer',
    },
  });

describe('index', () => {
  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });

  test('head', () => {
    expect(index.head()).toBeTruthy();
  });
});
