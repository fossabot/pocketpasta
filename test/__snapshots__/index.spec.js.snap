// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index renders properly 1`] = `
"<div label=\\"Job Title\\" value=\\"Full Stack Web Developer\\"><section><v-parallax src=\\"https://res.cloudinary.com/pocketpasta/image/upload/c_fit,f_auto,q_auto,w_2200/v1559982334/hero_rko6us.jpg\\" height=\\"600\\"><v-layout column=\\"\\" align-center=\\"\\" justify-center=\\"\\" class=\\"white--text\\"><b-img-lazy src=\\"200\\" alt=\\"Pasta on a sheet of paper\\" height=\\"200\\" width=\\"200\\"></b-img-lazy> <h1 class=\\"brand white--text mb-2 text-xs-center\\">
          PocketPasta
        </h1> <div class=\\"subheading mb-3 text-xs-center\\">
          landing.banner.subheading
        </div> <credit-stub style=\\"position: absolute; right: 0px; bottom: 0px;\\"></credit-stub></v-layout></v-parallax></section> <section><v-container grid-list-xl=\\"\\"><v-layout row=\\"\\" wrap=\\"\\" justify-center=\\"\\" class=\\"my-5\\"><v-flex xs12=\\"\\" sm4=\\"\\"><v-card class=\\"elevation-0 transparent\\"><v-card-title primary-title=\\"\\" class=\\"layout justify-center\\"><div class=\\"headline\\">landing.info.heading</div></v-card-title></v-card></v-flex> <v-flex xs12=\\"\\" sm4=\\"\\" offset-sm1=\\"\\"><v-card class=\\"elevation-0 transparent\\"><v-card-title primary-title=\\"\\" class=\\"layout justify-center\\"><div class=\\"headline\\">landing.contact.heading</div></v-card-title> <v-list class=\\"transparent\\"><v-list-tile><v-list-tile-action><v-icon class=\\"blue--text text--lighten-2\\">place</v-icon></v-list-tile-action> <v-list-tile-content><v-list-tile-title>landing.contact.location</v-list-tile-title></v-list-tile-content></v-list-tile> <v-list-tile><v-list-tile-action><v-icon class=\\"blue--text text--lighten-2\\">email</v-icon></v-list-tile-action> <v-list-tile-content><v-list-tile-title><a href=\\"mailto:landing.contact.email\\">landing.contact.email</a></v-list-tile-title></v-list-tile-content></v-list-tile></v-list></v-card></v-flex></v-layout></v-container></section></div>"
`;
